src/js/fotorama.js

```diff
@@ -1231,6 +1231,9 @@ jQuery.Fotorama = function ($fotorama, opts) {
   that.resize = function (options) {
     if (!data) return this;
 
+    // don't process resize for hidden elements
+    if ($wrap.is(':hidden')) return this;
```

Comment:
A lot of "if".
It can be added [here](https://github.com/artpolikarpov/fotorama/pull/470/files#diff-a8ba3933459ba275512d8b6c005bea21R1232)